src/utils/getMetadata.js

```diff
@@ -80,7 +80,7 @@ function firstImageUrlForAttributes(obj, attributes) {
     if (url) return;
 
     const binary = obj.get(attribute);
-    if (isVideoObj(binary)) return;
+    if (!isImageObj(binary)) return;
```

Comment:
* [ ] FYI we could optimistically name the const `image` like in SearchResultItem